core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -98,6 +84,14 @@ private[spark] class TaskSetManager(
   var totalResultSize = 0L
   var calculatedTasks = 0
 
+  val taskSetBlacklistOpt: Option[TaskSetBlacklist] = {
```

Comment:
Yeah I've seen this naming elsewhere (and agree about making things simpler when actually using the ariable) but found it particularly confusing here.  Maybe then call it taskSetBlacklistHelperOpt?  That's verbose but resolves my confusion about whether it's an option in the sense of being a config variable, rather than being a scala Option